com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -72,6 +72,10 @@ public class NetworkingManager : MonoBehaviour, INetworkUpdateSystem
 #endif
         public RpcQueueContainer rpcQueueContainer { get; private set; }
 
+        public delegate void PerformanceDataEventHandler(PerformanceTickData profilerData);
+
+        public event PerformanceDataEventHandler OnPerformanceDataEvent;
```

Comment:
`#if UNITY_EDITOR || DEVELOPMENT_BUILD` maybe? I wonder how are we planning to strip-out performance profiling stuff for non-Editor, non-Development targets, maybe I'm missing something?

Reply:
For performance reporting, I don't think we want to. The profiler itself should be stripped out in non-profiler but the data output itself is potentially relevant for end-user tools at runtime (for example, the net stats overlay). That said, users may still want to strip it out for performance reasons, but I think that should be a runtime check

Reply:
I'm doing 1-to-1 parity right now. so since these weren't previously in an #if then i'm not doing that in this PR. 

Ideally we could ifdef all instances of this class if we want.